pandas/core/arrays/interval.py

```diff
@@ -500,8 +500,10 @@ def __getitem__(self, value):
 
         # scalar
         if not isinstance(left, ABCIndexClass):
-            if isna(left):
+            if is_scalar(left) and isna(left):
                 return self._fill_value
+            if np.ndim(left) > 1:
```

Comment:
Can you clarify what this case is? Does this raise a warning?

Reply:
I'll add a comment.

It will issue a warning a few lines earlier when we do `left = self.left[value]`.

Looking at this more closely, we might want to zip the columns rather than stack them

Reply:
And what's being returned here? An ndarray of intervals?

Reply:
@jschendel thoughts on what should be returned here?

Reply:
it looks like we can just disallow this entirely

Reply:
Yeah, I'd say disallow it entirely.